### ISSUE TITLE
db conversion error

### ISSUE THREAD

**Issue by georg-e** [viktb/asnlookup#1]
Hey,
thanks for this awsome tool. 
Working with `.mrt` files works quite well. but actually i have expierienced issues when converting new asn dumps to the `.db` format. source from http://archive.routeviews.org/ or https://data.ris.ripe.net/rrc00/.
Converting historic dumps e.g. from may 2021 works.
Maybe we can find the issue. I just have expierienced it. Unfortunately haven't looked deeper into it.

Kind regards.

```
bzcat rib.*.bz2 | ./asnlookup-utils convert --input - --output test.db
2023/05/15 10:32:03 Failed to import: failed to parse MRT record: unknown BGP path attribute type code: 35
```

**Comment by viktb** [comment]
Hi @georg-e !
Seems like the [library](https://github.com/kaorimatz/go-mrt) used for parsing MRT doesn't support that BGP path attribute type. 
https://github.com/kaorimatz/go-mrt/blob/aa11f3646f93c0ecf3d4b52348a428f95e7b10c6/bgp.go#L268
I will look into a solution to this soon, hopefully in the coming days.

**Comment by viktb** [comment]
Fixed in [v0.1.1](https://github.com/banviktor/asnlookup/releases/tag/v0.1.1)
